.github/workflows/publish.yml

```diff
@@ -0,0 +1,154 @@
+name: Publish binaries
+
+on:
+  push:
+    tags:
+    - '*'
+  schedule:
+  - cron: '0 21 * * *'
```

Comment:
We still have to fix this for OpenSFX, but please distribute the time we build stuff on. Our website can only generate 1 version every ~3 minutes, so if all Open**X start at the same time, that might collide :P OpenGFX and OpenSFX atm start at the same time, but OpenGFX takes significantly longer, so that isn't an issue -yet- :D

Rather later than earlier btw, offsets of 15 minutes will be fine.

Reply:
Aha, yes, no problem :)